data/hai/hai reveal 1 intro.txt

```diff
@@ -73,28 +73,36 @@ mission "Event: Pug Media Coverage"
 					goto nobody
 				`	"I think you both need to shave."`
 					goto scowl
-			`	The right twin rolls his eyes while the left twitches his mustache awkwardly and responds. "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting about defending against unknown dangers at the fringes of human space. Non-stop! Don't we have enough immediate problems without imagining new horrors?"`
+			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`
+			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting against Admiral uh..." He snaps his fingers trying to remember the name.`
+			`	"Admiral Danforth," says the right twin.`
+			`	"Right. Danforth. This Councilor wants Danforth to defend against unknown dangers at the fringes of human space. Don't we have enough immediate problems without imagining new horrors?"`
 			`	"Like terrorists nuking us," says the right twin.`
 			choice
+				`	"The Navy is skilled and honorable. I trust them to protect us."`
+					goto eyes
 				`	"No, don't worry about that. I ended the bomb threat."`
-				`	"There's a lot of dangers out there, but don't worry. I'll be there to fight them off."`
 				`	"Maybe you're right."`
 					goto shrug
 				`	"All we can do is get on with our lives, right?"`
 					goto shrug
 				`	"If you don't shave soon, people will think you're wearing masks."`
 					goto scowl
 			label nobody
-			`	The right twin says, "Who are you, Admiral Danforth?" The left one laughs and adds, "Did the Navy fire you or are you here on holiday?" Both of them laugh for a few seconds before you can respond.`
+			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
+			`	Both of them laugh for a few seconds before you can respond.`
 			choice
-				`	"No. I'm the famous Captain <first> <last>!"`
-				`	"You don't recognize me?"`
-					goto shrug
+				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
 				`	"Never mind."`
 					goto shrug
 				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
 					goto scowl
-			`	Both twins look confused and say in unison, "Who?" They shrug and walk away, leaving you to wonder whether your doings are hidden in a cloud of government-enforced secrecy, or if some people have already forgotten about you.`
+			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`
```

Comment:
I quite like this change. Really sticks the player down to earth here.

Reply:
That's my main goal in this mission rewrite. Your ego is at an all-time high, and you think you're the most important person in the universe. In every bar and spaceport, everyone should know your name. Now, you find out that people have been living lives without constant news of Captain Last. It gives you a big dose of reality when you need it the most.